runtime-modules/content/src/lib.rs

```diff
@@ -189,8 +189,16 @@ pub struct ChannelCategoryUpdateParameters {
 /// If a channel is deleted, all videos, playlists and series will also be deleted.
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 #[derive(Encode, Decode, Default, Clone, PartialEq, Eq, Debug)]
-pub struct ChannelRecord<MemberId, CuratorGroupId, DAOId, AccountId, VideoId, PlaylistId, SeriesId>
-{
+pub struct ChannelRecord<
+    MemberId,
+    CuratorGroupId,
+    DAOId,
+    AccountId,
+    VideoId,
+    PlaylistId,
+    SeriesId,
+    CollaboratorsSetType,
```

Comment:
You did not address why we need  `CollaboratorsSetType`, rather than just using `BTreeSet<>` inlined? It's fine if you disagree, but then you should provide a rationale that rebuts my review comment as a reply to the prior comment.

Reply:
OK sorry about that I forgot to mention. I used `CollaboratorSetType` because BTree set requires a key that implements
the Ord trait, This will produce a long list of errors. This is related to the previous [ssue](https://github.com/Joystream/joystream/issues/2492#issuecomment-868261879).

Reply:
But I don't think the implication of this issue is that you have to use this solution, because we are not needing to do this many other places where we do have BTreeSet, right @shamil-gadelshin  ?

Reply:
We can use `MemberId: Ord` as a generic type requirement for the type with the `BTreeSet`. That could solve the issue.